hw1/hw1.js

```diff
@@ -0,0 +1,97 @@
+//Задание 1
+const loop = (times = 0, callback = null) => {
+	if (!callback || times === 0 || +times === NaN) {
+		return;
+	}
+	while(times >= 0) {
```

Comment:
```suggestion
	while(times > 0) {
```

в противном случае цикл будет срабатывать больше ожидаемого. loop(1, ...) сработает дважды, хоть мы и уточняем единичное выполнение